boards/cats_rev1Pro/src/control/kalman_filter.h

```diff
@@ -22,23 +22,21 @@
 #include "util/types.h"
 #include "util/error_handler.h"
 
-#define STD_NOISE_BARO   9.0f    // From data analysis: 2.6f
-#define STD_NOISE_IMU    0.004f  // From data analysis: 0.004f
-#define STD_NOISE_OFFSET 0.000001f
+#define STD_NOISE_BARO         900000.0f  // From data analysis: 2.6f
+#define STD_NOISE_BARO_INITIAL 9.0f       // From data analysis: 2.6f
+#define STD_NOISE_IMU          0.004f     // From data analysis: 0.004f
+#define STD_NOISE_OFFSET       0.000001f
```

Comment:
do these values have units? in the baro comments it says 2.6f for both but the values are different?